src/translations/et/common.json

```diff
@@ -22,6 +22,8 @@
   "notifications.idle-chat-notification": "Kas soovite vestlust jätkata?",
   "notifications.ask-contact-information": "Kas soovite jätta enda kontaktandmed, et klienditeenindaja saaks ise ühendust võtta?",
   "notifications.human-took-over": "Klienditeenindaja võttis vestluse üle",
+  "notifications.ask-to-forward-to-csa": "Paluti vestlus klienditeenindajale üle kanda",
+  "notifications.forwarded-to-backoffice": "Vestlus suunatakse tagasi kontorisse",
```

Comment:
Could you please double-check the case for these two? Similarly to my [comment here](https://github.com/buerokratt/Training-Module/pull/728#discussion_r1865951631). Why is `forwarded-to-backoffice` in kebab case here but in snake case in the Training Module PR?